src/core/actor/Mmm1dgpuForce_cuda.cu

```diff
@@ -356,13 +356,13 @@ __global__ void forcesKernel(const __restrict__ mmm1dgpu_real *r, const __restri
 			sum_r *= sqpow(uz);
 			sum_z *= sqpow(uz);
 
-			sum_r += rxy*cbpow(rsqrt(rxy2+pow(z,2)));
-			sum_r += rxy*cbpow(rsqrt(rxy2+pow(z+boxz,2)));
-			sum_r += rxy*cbpow(rsqrt(rxy2+pow(z-boxz,2)));
+			sum_r += rxy*cbpow(rsqrt(rxy2+sqpow(2)));
```

Comment:
This is wrong and should fail the test case. Please change this line to `sum_r += rxy*cbpow(rsqrt(rxy2+sqpow(z)));`